src/flows/components/PresetFlows.tsx

```diff
@@ -17,17 +17,53 @@ const PresetMap = {
   'Blank Notebook': '/notebook/from/blank',
 }
 
-const PresetFlows: FC = () => {
+interface Props {
+  buttonMode?: boolean
+}
+const PresetFlows: FC<Props> = ({buttonMode}) => {
   const history = useHistory()
-  return (
+  return buttonMode ? (
+    <Grid>
+      <Grid.Row>
+        <Grid.Column
+          widthXS={Columns.Twelve}
+          widthSM={Columns.Eight}
+          widthMD={Columns.Ten}
+        >
+          <div className="flows-index--presetList buttonModeList">
+            {Object.keys(PresetMap).map((p: string, idx: number) => (
+              <div key={p} className="flows-index--presetButtons">
+                {idx === 0 ? (
+                  <Button
+                    color={ComponentColor.Primary}
+                    icon={IconFont.Plus}
+                    text={p}
+                    onClick={() => history.push(PresetMap[p])}
+                    className="flows-preset--buttonmode"
+                  ></Button>
+                ) : (
+                  <Button
+                    text={p}
+                    color={ComponentColor.Tertiary}
+                    onClick={() => history.push(PresetMap[p])}
+                    className="flows-preset--buttonmode"
+                  ></Button>
+                )}
+              </div>
+            ))}
+          </div>
+        </Grid.Column>
+      </Grid.Row>
+    </Grid>
+  ) : (
     <Grid>
       <Grid.Row>
         <Grid.Column
           widthXS={Columns.Twelve}
           widthSM={Columns.Eight}
           widthMD={Columns.Ten}
         >
-          <div className="flows-index--presetContainer">
+          <div id="presetContainer" className="flows-index--presetContainer">
```

Comment:
dont use ids in react, it causes weird shit to happen. if you need to access something directly, use a ref

Reply:
this was a floating id, thanks for pointing this out, removing it. It wasn't actually doing anything